go-controller/pkg/ovn/base_network_controller_namespace.go

```diff
@@ -357,6 +357,9 @@ func (bnc *BaseNetworkController) getAllNamespacePodAddresses(ns string) []net.I
 	} else {
 		ips = make([]net.IP, 0, len(existingPods))
 		for _, pod := range existingPods {
+			if !bnc.doesNetworkRequireIPAM() {
```

Comment:
you can just return an empty list with this condition at the very beginning of this function right? I mean no need to iterate over pods?

Reply:
Done.

Reply:
hehe never underestimate proper rest! thanks for the fast fix